### ISSUE TITLE
Cannot find module 'C:\Repos\docs\index-build'

### ISSUE THREAD

**Issue by sesispla** [docarys/docarys#14]
## Description

Error when using docarys on Windows

### Expected behavior

When invoking build or serve, docarys should execute

### Actual behavior

The following error is throw in Windows:

```ps
Windows PowerShell
Copyright (C) 2016 Microsoft Corporation. Todos los derechos reservados.

PS C:\Repos\docs> docarys bulid
PS C:\Repos\docs> docarys build
module.js:472
    throw err;
    ^

Error: Cannot find module 'C:\Repos\docs\index-build'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:425:7)
    at startup (bootstrap_node.js:146:9)
    at bootstrap_node.js:540:3
```

### Steps to reproduce the bug

1. Create an empty project with `yo docarys` on Windows
2. Execute `docarys build` or `docarys serve`

### Package versions

* Node: `node --version` v7.7.2
* Docarys: `docarys --version` v1.0.7
* Material: `npm view docarys-material version` v1.0.4
* Generator: `npm view generator-docarys version` v1.0.2

### Project configuration

``` yaml
site_name: docs
theme: material
extra:
  feature:
    tabs: false
  palette:
    primary: lime
    accent: indigo
pages:
  Home: index.md
```

### System information

* OS: Windows 10 Enterprise (1067, bUILD 14393.953)

**Comment by sesispla** [comment]
@pabravil thank you for the report.

**Comment by sesispla** [comment]
It looks like only happens in Windows, since our Travis-CI build and development environment (MacOS) worked properly.

**Comment by sesispla** [comment]
Having a fast look, it sounds to something related with the Commander module we use to handle command-line arguments + Windows OS

**Comment by sesispla** [comment]
Fixed in v1.0.8
